tests/welcomePage.spec.js

```diff
@@ -0,0 +1,61 @@
+import {test, expect} from "@playwright/test";
+
+test.describe('Registration form verification', ()=>{
+
+    test('Positive registration', async({page})=>{
+        await page.goto('/')
+        const signUpButton = page.locator('.hero-descriptor_btn')
+        await signUpButton.click()
+        const name = page.locator('[id="signupName"]')
+        const lastName = page.locator('[id="signupLastName"]')
+        const email = page.locator('[id="signupEmail"]')
+        const password = page.locator('[id="signupPassword"]')
+        const reEnterPassword = page.locator('[id="signupRepeatPassword"]')
+
+        await name.fill('Myloslava')
+        await lastName.fill('Martynova')
+        await email.fill('aqa-malina@mailinator.com')
+        await password.fill('QWEasdqe11')
+        await reEnterPassword.fill('QWEasdqe11')
+
+        const registerButton = page.locator('div button.btn-primary:nth-child(1)')
+        await registerButton.click()
+    })
+
+    test('Verify error messages', async({page})=>{
+        await page.goto('/')
+        const signUpButton = page.locator('.hero-descriptor_btn')
+        await signUpButton.click()
+        const name = page.locator('[id="signupName"]')
+        const lastName = page.locator('[id="signupLastName"]')
+        const email = page.locator('[id="signupEmail"]')
+        const password = page.locator('[id="signupPassword"]')
+        const reEnterPassword = page.locator('[id="signupRepeatPassword"]')
+
+        await name.click()
+        await lastName.click()
+        const errorName = page.locator('.invalid-feedback').nth(0)
+        await errorName.isVisible()
```

Comment:
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible

Reply:
it does not perform any assertion, it just returns true | false
```
await errorName.isVisible()
```